src/pyclvm/instance/command.py

```diff
@@ -40,11 +40,11 @@ def _execute_azure(
     instance_name: str, instance: AzureRemoteShellProxy, **kwargs
 ) -> None:
     print(f"Starting {instance_name} ...")
-    instance.start()
+    instance.start(wait=False)
     print(f"{instance_name} is running")
     print(f"Working {instance_name} ...")
 
-    instance.execute((f"cd $HOME && {kwargs.get('script')}",), **kwargs)
+    instance.execute((kwargs.get("script"),), **kwargs)
```

Comment:
This change will cause the commands to run from `bin` folder. I strongly support to keep previous version here. Explicitly changed directory to execute all the commands from HOME folder.